weather_stats/weatherstatsapi/weather/openweather/client.py

```diff
@@ -0,0 +1,54 @@
+from types import TracebackType
+from typing import Optional, Type
+
+from aiohttp import ClientSession
+from pydantic import ValidationError
+
+from weather.openweather.exceptions import OpenWeatherError
+from weather.openweather.schemas import GetWeatherData, GetWeatherParams
+from weather.openweather.settings import OPEN_WEATHER_SETTINGS
+
+
+class OpenWeatherClient:
+
+    _session: ClientSession
```

Comment:
As this field looks like it exists only for sake of tests, [aioresponses](https://github.com/pnuckowski/aioresponses) package could be used to limit that kind of code pollution.

Reply:
It defines that field like this will exists in object of this class upon proper instillation. 

Having a session on instance level allows user to use the same session for multiple requests.